mypy/build.py

```diff
@@ -1978,17 +1978,17 @@ def load_fine_grained_deps(self) -> Dict[str, Set[str]]:
     def load_tree(self, temporary: bool = False) -> None:
         assert self.meta is not None, "Internal error: this method must be called only" \
                                       " for cached modules"
+
+        data = _load_json_file(self.meta.data_json, self.manager, "Load tree ",
+                               "Could not load tree: ")
+        if data is None:
+            return None
+
         t0 = time.time()
-        raw = self.manager.metastore.read(self.meta.data_json)
-        t1 = time.time()
-        data = json.loads(raw)
-        t2 = time.time()
         # TODO: Assert data file wasn't changed.
         self.tree = MypyFile.deserialize(data)
-        t3 = time.time()
-        self.manager.add_stats(data_read_time=t1 - t0,
-                               data_json_load_time=t2 - t1,
-                               deserialize_time=t3 - t2)
+        t1 = time.time()
+        self.manager.add_stats(deserialize_time=t1 - t0)
```

Comment:
Could you also record the time spent in `_load_json_file` here?

Reply:
Actually, `_load_json_file` does that internally. Maybe we can make it record `data_json_load_time` internally? I'd need to take a closer look at how these timings are being used, but I suspect it's not too important to keep them stable.

Reply:
Yeah, I can tweak it to add that metric!